0059-reduce-xor-filter-fees.md

```diff
@@ -0,0 +1,68 @@
+# HIP-59 Reduce XOR filter fees
+
+- Author(s): [@macpie](https://github.com/macpie), [@jdgemm](https://github.com/jdgemm)
+- Start Date: 2022-04-19
+- Category: Technical
+- Original HIP PR: [https://github.com/helium/blockchain-core/pull/1303](https://github.com/helium/blockchain-core/pull/1303)
+- Tracking Issue: [https://github.com/helium/HIP/issues/391](https://github.com/helium/HIP/issues/391)
+- Status: In Discussion
+
+# Summary
+
+Helium community members that host a Router instance use on-chain XOR filters to  process traffic from known devices in order to  significantly improve performance. This is an integrated component of a Router instance, and allows Routers and Packet Purchasers on the network to avoid spending resources on join packets from unknown devices. 
```

Comment:
```suggestion
Helium community members who host a Router instance use on-chain XOR filters to  process traffic from known devices in order to significantly improve performance. This is an integrated component of a Router instance, and allows Routers and Packet Purchasers on the network to avoid spending resources on join packets from unknown devices. 
```